### PR TITLE
feat: allow some staffworkflow fields to be blank

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** `scorer_id`, `grading_completed_at`, `grading_started_at`, `cancelled_at`, and `assessment` should be allowed to be blank

JIRA: [AU-428](https://openedx.atlassian.net/browse/AU-428)

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

Install ORA locally and navigate to https://internal.courses.edx.org/admin/assessment/staffworkflow/ and choose an unscored workflow. click save without changing anything. It should save correctly! set one of the datetime values and save, then clear and save. both should work.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by nsprenkle — APPROVED** [review]
LGTM 👍
